https-did/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="me.uport.sdk.https_did" />
```

Comment:
the packagename should not contain underscores.
`me.uport.sdk.httpsdid` should be fine